USAGE.md

```diff
@@ -69,15 +69,15 @@
   <!-- M -->
   <!-- N -->
 - [Faker.NaiveDateTime](lib/faker/naivedatetime.ex)
-- [Faker.Name](lib/faker/name.ex)
-- [Faker.Name.En](lib/faker/name/en.ex)
-- [Faker.Name.Es](lib/faker/name/es.ex)
-- [Faker.Name.Hy](lib/faker/name/hy.ex)
-- [Faker.Name.It](lib/faker/name/it.ex)
-- [Faker.Name.PtBr](lib/faker/name/pt_br.ex)
 - [Faker.Nato](lib/faker/nato.ex)
   <!-- O -->
   <!-- P -->
+- [Faker.Person](lib/faker/name.ex)
+- [Faker.Person.En](lib/faker/name/en.ex)
+- [Faker.Person.Es](lib/faker/name/es.ex)
```

Comment:
I wonder if we should rename the folder name to `person`.

```suggestion
- [Faker.Person.Es](lib/faker/person/es.ex)
```

Reply:
Good call!